README.md

```diff
@@ -46,6 +46,19 @@ Community Toolbox is built using a Grunt task from the source files in `/src/`,
 
 API calls are based on: https://www.npmjs.com/package/github-api-simple
 
+## Install Instructions
+* Clone or download the repo. into any fresh temporary folder.
```

Comment:
Could you offer the git command in full here? Like `git clone ...`

Thanks!!!

Reply:
Sure!